app/models/external-storage-service.ts

```diff
@@ -2,10 +2,19 @@ import { attr } from '@ember-data/model';
 
 import OsfModel from './osf-model';
 
+export enum CredentialsFormat {
+    OAUTH = 'oauth',
+    OAUTH2 = 'oauth2',
+    URL_USERNAME_PASSWORD = 'url_username_password',
+    REPO_TOKEN = 'repo_token',
+    ACCESS_SECRET_KEYS = 'access_secret_keys',
```

Comment:
There is also a `username_password` type. Six total including both oauths.

Reply:
Ahh, I missed that. Which one uses this type? I will add it to the mirage fixture as well.

Reply:
Looks like it's Boa.